io.mjml.mjml-app.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: io.mjml.mjml-app
+base: org.electronjs.Electron2.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: mjml-app
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: mjml-app
+    buildsystem: simple
+    build-commands:
+      - install -Dt /app/bin apply_extra mjml-app
+      - install -Dm644 -t /app/share/metainfo io.mjml.mjml-app.appdata.xml
+      - install -Dm644 -t /app/share/applications io.mjml.mjml-app.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/128x128/apps io.mjml.mjml-app.png
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - mkdir _tmp
+          - mv *.AppImage _tmp
+          - cd _tmp
+          - chmod +x *.AppImage
+          - ./*.AppImage --appimage-extract
+          - mv squashfs-root ../mjml-app
+          - cd ..
+          - rm -rf _tmp
+          - chmod -R 755 mjml-app
+      - type: script
+        dest-filename: mjml-app
+        commands:
+          - exec /app/extra/mjml-app/app/mjml-app "$@"
```

Comment:
This one is not needed, you could just link the mjml-app to `/app/bin` instead of using this.